organization/_posts/2017-10-18-packaging.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "HSF Packaging Group Meeting #12, October 18, 2017"
+layout: default
+---
+
+# HSF Packaging Group Meeting #12, October 18, 2017
+
+#### *Present*: Graeme Stewart, Benedikt Hegner, Liz Sexton-Kennedy, Guilherme Amadio, Pere Mato, Patricia Mendez Lorenzo, Javier Cervantes Villanueva, Marco Clemencic, Ben Couturier, Ben Morgan, Christopher Green, Jim Amundson, Patrick Gartung, David Lange, Emil Obreshkov, Shahzad Muzaffar, Oana Boeriu, Joel Closier
+
+#### [Indico Agenda and Presentations](https://indico.cern.ch/event/672745/){:target="_hsf_packaging_12"}
+
+## Introduction
+
+It was agreed that containers are an important topic for the group.
+
+## Progress Reports
+
+### Spack
+
+PGP signing is much more important for HPCs. HEP tends to have a trusted build chain managed by the same team, from checkout to CVMFS deployment.
+
+Relocatable build caches added to Spack, which was one of the missing features identified during the initial review.
+
+Additional topic on moving CMSSW from scram to CMake. Development environment is still missing.
+
+Pere: questions on relocation - how is that done? A: relative RPATH and text files get a path replacement at install time (rewriting the original file).
+
+Guilherme: what if the relocated RPATH is bigger than the original one? A: On MacOS you can set size. On Linux patchelf allows to use a bigger one. If path bigger than place reserved, you append on the end. Thus not a problem in practice.
+
+RPATH vs. RUNPATH discussion: for a development environment RUNPATH would be strongly favoured.
+
+### SpackDev
+
+Presented how to set up a development environment with SpackDev. There is an added tool to ease building against system packages.
+
+Graeme: Can the working package set be updated? A: Not at the moment, but this feature is envisaged.
+
+What’s the definition of a "package"? SpackDev for packages as standalone externals or experiments' notion of packages as a subset of the code in the experiment application. A: This is not entirely scoped out yet. *Agreed on follow-up discussion on that.*
+
+### Portage
+
+Presentation: clarified that prefixes can depend on each other, so complete environments can be built out of pieces. 
```

Comment:
I am not too sure about it. To me that sounded more like the layered approach of docker containers rather than a tree like combination of pieces. I guess that needs follow up.

Reply:
That uncertainty should be reflected in the new version